### PR TITLE
Loadout/Preferences Tweaks II

### PR DESCRIPTION
**Pull request by MarkSuckerberg**
## About The Pull Request
Does a few changes I think the playerbase would enjoy, such as linking loadouts to a character slot instead of a client. Also fixes a few things, and adds rudimentary support for departmental items such as berets, which I have made actually work.

image
image
*both images taken with the same loadout item equipped.*

## Why It's Good For The Game
People'd enjoy this

## Changelog
:cl:
add: Departmental loadout support (see: berets)
add: Loadouts now linked to character slots instead of clients
tweak: You can now have more than one item equipped from the loadout if you have a slot for it
fix: The toggle loadout button in the preferences menu now works again
fix: Some berets had messed up sprites, which have been fixed
/:cl:

### CODE REVIEW

File: waspstation/code/modules/clothing/head/berets.dm [hunk 1]
```diff
@@ -50,6 +50,11 @@
 	desc =  "A standard grey beret. Why an assistant would need a beret is unknown."
 	icon_state = "beret_grey"
 
+/obj/item/clothing/head/beret/puce
+	name = "strange beret"
+	desc =  "You're not sure what to make of this."
+	icon_state = "beret_PUCE"
```
Comment (Superyodeler) [inline waspstation/code/modules/clothing/head/berets.dm:56]: P U C E

Comment (MarkSuckerberg) [reply waspstation/code/modules/clothing/head/berets.dm:56]: Puce.

**Comment by Superyodeler** [comment]
This should probably be testmerged

**Comment by Maldaris** [comment]
> This should probably be testmerged

On the condition the backup save system works appropriately and doesn't obliterate people's loadouts/static characters/etc.

**Comment by MarkSuckerberg** [comment]
I'll test it thoroughly again to make sure it does not cause any issues with saves, but from initial testing nothing was lost.

**Review by Maldaris — APPROVED** [review]

**Pull request merged**